src/python/aps.py

```diff
@@ -0,0 +1,345 @@
+import numpy as np
+import numpy.ctypeslib as npct
+from ctypes import c_int, c_uint, c_ulong, c_ulonglong, c_float, c_double, c_char, c_char_p, addressof, create_string_buffer, byref, POINTER
+
+libaps2 = npct.load_library("libaps2", ".")
```

Comment:
Let's assume library is in `build` for now. Also have you tested dependency loading on Windows?